### ISSUE TITLE
Distinguish push'd packages and mirrored ones

### ISSUE THREAD

**Issue by RachelDavids** [bagetter/BaGetter#187]
### Is your feature request related to a problem? Please describe.
I'm trying to clear out all mirrored packages to reduce the overhead on my hard disk  From this point onwards only the latest packages would be added.

### Describe the solution you'd like
A way to distinguish between explicitly pushed packages and those added via mirroring.

### Describe alternatives you've considered
A manual process of elimination, by checking package ids on the mirror. (Nuget.org)

### Additional context
Using SQL Server back-end does make searching for  package detail easy, especially when wanting everything that does NOT match a given criteria. However, these is no easy way to find the packages I want to delete.

**Comment by Regenhardt** [comment]
We have multiple conversations about auto cleanup features. One of them would fit your use case: A setting that would limit the number of versions to keep for each mirrored package. That way, you could set that to a low number and automatically have older packages removed when newer ones are added. This might also include a flag indicating the origin of each package.

**Comment by viceice** [comment]
Yeah, would be nice to have a differrent value `MaxVersionsPerPackage` for mirror packages.

https://www.bagetter.com/docs/configuration#enable-package-auto-deletion
